### DISCUSSION TITLE
Old delisted ranked maps showing an update prompt

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19357] [Q&A] [answered]
![sssss](https://user-images.githubusercontent.com/74463310/180665383-265e8dae-4d2d-42ae-a8e9-078961c85580.jpg)
![osu_2022-07-24_22-50-33](https://user-images.githubusercontent.com/74463310/180665386-45910d81-1d1e-448d-885d-026d87ea4cbe.jpg)
pretty confusing, maybe something could be done to stop delisted maps prompting updates all together, but i cant imagine why they are doing so in the first place

worth noting i have 200 ish delisted maps and only this one currently prompts me for update, not a generalised issue

**Comment by peppy** [comment, marked as answer]
This looks like correct behaviour. The update button will always show if beatmaps don't match the local version, which they likely don't in this case.

**Reply by mk56-spn** [reply]
How is that possible though if it's the version that allows score submission? Didn't know that was possible

**Reply by peppy** [reply]
Not sure. Keep in mind that lazer doesn't check hashes for score submission just yet, so you may actually be unknowningly doing bad. Tracking this at https://github.com/ppy/osu-web/pull/9137.

**Reply by Feodor0090** [reply]
Right now it will throw exception due to 404 on beatmap's file.
[runtime (5).log (line 139)](https://github.com/ppy/osu/files/9658224/runtime.5.log)
This should _at least_ have correct message for user. And i'm still insisting that this is not a correct behavior - if there is nothing to update to, it should not be prompted (both for deleted and DMCAed maps). A message must be shown instead of clickable button.

About hashes matching - for me (and my friend who initially found this) levan polkka from https://osu.ppy.sh/community/forums/topics/49310?n=1 shows update button right after importing, which is kinda strange.
